src/azul/compliance/fedramp_inventory_service.py

```diff
@@ -468,7 +480,7 @@ def get_inventory(self, resources: Iterable[JSON]):
                   f'{resource_counts[resource_type]:>15d}'
                   f'{row_counts[resource_type]:>10d}')
 
-        return inventory
+        return sum((rows_by_mapper[mapper] for mapper in self._mappers), start=[])
```

Comment:
```suggestion
        return flatten(rows_by_mapper[mapper] for mapper in self._mappers)
```

With `flatten` from more_itertools. If that works.